test/fixtures.ts

```diff
@@ -445,6 +445,93 @@ export default {
       ],
     },
   ],
+  cip3Icarus: {
```

Comment:
I'd suggest adding also a 24-word mnemonic, to ensure we didn't accidentally deviate there (as Trezor did). Backstory - trezor implementation of mnemonic to entropy for cardano had a bug that manifested only for 24 word mnemonics and it caused trezor to deviate

so I'd cover that explicitly with a testcase

https://github.com/cardano-foundation/CIPs/blob/master/CIP-0003/Icarus.md#trezor

Reply:
link to testcase for trezor for 24 word mnemonic: https://github.com/trezor/trezor-firmware/pull/1388/files#diff-17c19a8d23037f23062138997edf774fde43ab9dfbf7726d1f701edb6a2f133eR105

bear in mind that using the same mnemonic here should actually produce a different public key, because of the aforementioned bug/deviation (recommend cross-checking e.g. with NuFi by console.log when initiated to that mnemonic)